solutions/swb-reference/README_SINGLE_CLICK.md

```diff
@@ -0,0 +1,55 @@
+# SWBv2 Single-Click Deployment
+
+## Get main account CloudFormation template
+- Navigate here <TODO: Enter valid URL> to the SWBv2 Solutions Implementation page and download the Cloudformation template. 
+- Deploy this stack in your main AWS account by providing the necessary input parameters.
+
+### Breaking down your stack name
+While following along steps you encounter further, please note that since you're working with a pre-formed template the region short name is a random string. For example if your stack name is `swb-dev-bb5823` then:
+- `dev` is the stage name
+- `bb5823` is the region short name
+
+## Perform post-deployment steps
+Currently Service Workbench contains some steps that can only be performed upon the completion of the main account CloudFormation stack. Here are the final few steps to complete your main account setup:<br/>
+
+1. Create a Cloud9 environment in the main account and in the same region as your CloudFormation stack deployment
```

Comment:
Does the Cloud9 environment need a specific instance role so that it can call cf describe-stacks/ssm put-parameter?